File: abc/abc326/a.cpp
// https://atcoder.jp/contests/abc326/submissions/46992358

#include <bits/stdc++.h>
using namespace std;

int main() {
  int x, y;
  cin >> x >> y;

  if(x <= y && abs(y - x) <= 2) {
    cout << "Yes";
    return 0;
  }

  if(x >= y && abs(y - x) <= 3) {
    cout << "Yes";
    return 0;
  }

  cout << "No";
}

File: abc/abc326/a.java
// https://atcoder.jp/contests/abc326/submissions/47098989

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int x = stdIn.nextInt();
    int y = stdIn.nextInt();

    for(int i = x - 3; i < x + 3; i++) {
      if(i == y) {
        System.out.println("Yes");
        System.exit(0);
      }
    }

    System.out.println("No");
  }
}
